formal-resume/resume_english.tex

```diff
@@ -22,42 +22,42 @@
 
 \basicInfo{
   \email{Kenji.Mouri@outlook.com} \textperiodcentered\
-  \homepage{https://mouri.moe}
+  \homepage{https://mouri.moe} \textperiodcentered\
```

Comment:
Add consistent spacing between headers items

Reply:
I will follow the suggestion.